CMakeLists.txt

```diff
@@ -49,6 +49,11 @@ endif()
 
 message(STATUS "EmptyEpsilon Version = ${PROJECT_VERSION}")
 
+# Enable colored output with Ninja
+if(CMAKE_GENERATOR MATCHES "Ninja")
```

Comment:
Should also have a check to see if the compiler is GCC, else you could pass this option to msvc.

Reply:
Makes sense. Done.